practice/practice-02/src/sass/main.scss

```diff
@@ -1 +1,17 @@
+// Abstracts
+@import "./abstract/mixins";
+@import "./abstract/variables";
+
+//Base
```

Comment:
Missing space, pls correct all

Reply:
Do you mean missing space between lines 2 and 3?

Reply:
In current line